src/pl/devsmentoring/java/Module5/Exercise20.java

```diff
@@ -0,0 +1,50 @@
+package pl.devsmentoring.java.Module5;
+
+import java.util.*;
+
+public class Exercise20 {
+    public static void main(String[] args) {
+
+        List<String> colourList = Arrays.asList("zielony", "czerwony", "niebieski", "czarny",
+                "fioletowy", "granatowy", "niebieski",
+                "czarny", "czarny", "zielony", "cytrynowy", "granatowy",
+                "niebieski", "indygo", "zielony", "czerwony");
+
+        System.out.println("Original length of our colour list was: " + colourList.size());
+
+
+        Map<String, Integer> colorCounts = new HashMap<>();
+        for (String color : colourList) {
+            colorCounts.putIfAbsent(color, 0);
+            colorCounts.put(color, colorCounts.get(color) + 1);
```

Comment:
tutaj najpierw wrzucasz kolor, a później dodajesz do niego 1.

Przekształć to w taki sposób żeby najpierw dochodziło do instrukcji warunkowej gdzie sprawdzasz czy istnieje taki kolor w mapie. Jeśli nie istnieje - dodajesz go do mapy z wartością 1. Jeśli istnieje, inkrementujesz jego wartość